### PR TITLE
Document act usage to run GitHub action locally

### LINKED ISSUE #431: [REQUEST] Study possibility to run GitHub actions locally
**Is your feature request related to a problem? Please describe.**
This a split of #416 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Study possibility to run locally github actions
https://github.com/nektos/act

### PR DESCRIPTION
**Pull request by fabrice-v**
closes #431 

**Summary:**

This PR updates the documentation to describe how to run the end to end workflow locally in Docker via the `act` project
https://github.com/nektos/act

**Expected behavior:** 

Syncing the latest version of master and passing the command `act -j run-on-data`
Now works fine with the validator running on all datasets until the persist artifact steps...
image

.. which is a known issue of `act`
https://github.com/nektos/act/issues/329

The files in question can be manually collected inside the container. 
Datasets `.zip` files are present as well as validation reports `.json` in the `output` folder
image

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [ ] Format the title like "Fix #<issue_number> - <short description of fix and changes>" (for example - "Fix #1111 - Check for null value before using field")
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,6 +272,55 @@ $ ./gradlew check
...
 $ npm test
 ```
+
+## End to end
+
+### via `run-on-data` GitHub workflow job
+
+There is a way to locally execute the [`run-on-data` job of the end_to_end GitHub workflow](https://github.com/MobilityData/gtfs-validator/blob/24d58c8ee76af00aa3ab413b218b4a8e2cfafc4b/.github/workflows/end_to_end.yml#L9)
+
+You need Docker installed
```
Comment (lionel-nj) [inline README.md:282]: I couldn't add a suggestion. However this could be "You need Docked to be installed"

File: README.md [hunk 2]
```diff
@@ -269,6 +272,55 @@ $ ./gradlew check
...
+adapter		     mbta.zip
+application	     mixed_empty_full_gtfs_files.zip
+# cd output
+# ls
+MBTA__2020-10-26_08-51-29.211229.json
+MST__2020-10-26_08-51-39.835092.json
+Orange_County_Transportation_Authority__2020-10-26_08-52-19.024739.json
+```
+
+
```
Comment (lionel-nj) [inline README.md:319]: Do we need all these extra blank lines?

**Review by lionel-nj — APPROVED** [review]
LGTM! A few comments in-line though.

File: README.md (hunk 2 above)
Comment (fabrice-v) [reply README.md:335]: removing

**Pull request merged**